src/page-modules/assistant/assistant.module.css

```diff
@@ -7,6 +7,9 @@
   padding: 0 var(--spacings-medium);
   margin: 0 auto;
 }
+.homeLink__container > :last-child:not(:only-child) {
+  margin-bottom: var(--spacings-medium);
+}
```

Comment:
Is this better handled by using `gap` in `homeLink__container`?

Reply:
But.. Is homeLink__container in use here now? Seems like it is removed from usage in assistant layout

Reply:
Not in use here, so I've removed it. I think using `gap` on the container would result in extra space between the button and the first global message, which we don't want.